cantor-archive/src/main/java/com/salesforce/cantor/archive/file/EventsArchiverOnFile.java

```diff
@@ -7,18 +7,40 @@
 
 package com.salesforce.cantor.archive.file;
 
+import com.google.protobuf.ByteString;
 import com.salesforce.cantor.Events;
+import com.salesforce.cantor.archive.EventsChunk;
+import com.salesforce.cantor.common.EventsPreconditions;
 import com.salesforce.cantor.misc.archivable.EventsArchiver;
+import org.apache.commons.compress.archivers.ArchiveEntry;
+import org.apache.commons.compress.archivers.ArchiveInputStream;
+import org.apache.commons.compress.archivers.ArchiveOutputStream;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
+import java.nio.file.Files;
 import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import static com.salesforce.cantor.common.CommonPreconditions.checkArgument;
+import static com.salesforce.cantor.common.CommonPreconditions.checkNamespace;
 
 public class EventsArchiverOnFile extends AbstractBaseArchiverOnFile implements EventsArchiver {
     private static final Logger logger = LoggerFactory.getLogger(EventsArchiverOnFile.class);
     private static final String archivePathFormat = "/archive-events-%s-%d-%d";
+    private static final Pattern archiveRegexPattern = Pattern.compile("archive-events-.*-(?<start>\\d+)-(?<end>\\d+)");
+
+    private static final String FLAG_RESTORED = ".cantor-archive-restored";
```

Comment:
let's do dimension for this, and `s/FLAG_RESTORED/isRestoredFlag/` (camel case for other private statics too)

Reply:
Changed.